contrib/yaml/postgresql-16.yaml

```diff
@@ -0,0 +1,199 @@
+        - name: reads
+          type: counter
+          description: Number of read operations.
+        - name: read_time
+          type: counter
+          description: Total time spent on read operations in milliseconds.
+        - name: writes
+          type: counter
+          description: Number of write operations.
+        - name: write_time
+          type: counter
+          description: Total time spent on write operations in milliseconds.
+        - name: writebacks
+          type: counter
+          description: Number of writeback to permanent storage requests sent to kernel.
+        - name: writeback_time
+          type: counter
+          description: Total time spent on writeback operations in milliseconds.
+        - name: extends
+          type: counter
+          description: Number of relation extend operations.
+        - name: extend_time
+          type: counter
+          description: Total time spent on relation extend operations in milliseconds.
+        - name: op_bytes
+          type: gauge
+          description: Bytes per unit of I/O read, written or extended.
+        - name: hits
+          type: counter
+          description: The number of times a desired block was found in shared buffer.
+        - name: evictions
+          type: counter
+          description: The number of times a block has been written out from shared or local buffer in order to make it available for another use.
+        - name: reuses
+          type: counter
+          description: The number of times an existing buffer in a size-limited ring buffer outside of shared buffers was reused as part of an I/O operation.
+        - name: fsyncs
+          type: counter
+          description: Number of fsync calls.
+        - name: fsync_time
+          type: counter
+          description: Total time spent on fsync operations in milliseconds.
+    tag: pg_stat_io
+    collector: stat_io
+
+# stat_database_conflicts_information()
+  - queries:
+    - query: SELECT datname,
+                confl_tablespace,
+                confl_lock,
+                confl_snapshot,
+                confl_bufferpin,
+                confl_deadlock
+              FROM pg_stat_database_conflicts
+              WHERE datname IS NOT NULL
+              ORDER BY datname;
+      columns:
+        - name: database
+          type: label
+        - name: confl_tablespace
+          type: counter
+          description: pg_stat_database_conflicts_confl_tablespace
+        - name: confl_lock
+          type: counter
+          description: pg_stat_database_conflicts_confl_lock
+        - name: confl_snapshot
+          type: counter
+          description: pg_stat_database_conflicts_confl_snapshot
+        - name: confl_bufferpin
+          type: counter
+          description: pg_stat_database_conflicts_confl_bufferpin
+        - name: confl_deadlock
+          type: counter
+          description: pg_stat_database_conflicts_confl_deadlock
+      version: 10
```

Comment:
Better to do a comment section that lists all 10, like

```
#
# PostgreSQL 10
#
```

Reply:
I didn't quite get this, can you please explain this more?

I've provided alternate versions of queries that are available in lower versions along with the alternative that is supported on the current version.

Also, should I make this change in every file?